config/locales/en.yml

```diff
@@ -26,6 +26,8 @@ en:
       code: App-generated code
     confirmation:
       show_hdr: Create a Password
+    value_missing: This field is required
```

Comment:
should these all be grouped together? (the generic errors and the more specific ones)